contracts/TokenListManager.sol

```diff
@@ -125,6 +125,33 @@ contract TokenListManager is ITokenListManager, AccessControl {
         }
     }
 
+    // Function to get all whitelisted tokens for a given TokenType
+    function getAllWhitelistedTokens(TokenType tokenType) public view returns (address[] memory allTokens) {
```

Comment:
Why not reuse `getWhitelistedTokens` with a large enough `limit` (e.g. uint256.MAX) ?

Reply:
Solved, I use large enough limit to cover most cases.